pkg/config/types/generated_constants.go

```diff
@@ -72,6 +72,8 @@ const NodeComputeQueueExecutorBufferBackoffDuration = "Node.Compute.Queue.Execut
 const NodeComputeLogging = "Node.Compute.Logging"
 const NodeComputeLoggingLogRunningExecutionsInterval = "Node.Compute.Logging.LogRunningExecutionsInterval"
 const NodeRequester = "Node.Requester"
+const NodeRequesterJobDefaults = "Node.Requester.JobDefaults"
```

Comment:
As the commit message for this change calls out. These values were changed in https://github.com/bacalhau-project/bacalhau/pull/2791 but we forgot to regenerate the config constants for it. Addressing here.

Reply:
We should probably add a CI check for this..

Reply:
Added here: https://github.com/bacalhau-project/bacalhau/pull/2828